webapp/app.py

```diff
@@ -178,7 +170,36 @@ def getpassengers():
     passengers = cur.fetchall()
     cur.close()
     conn.close()
-    return flask.jsonify(passengers)
+
+##################################################################################
+
+# Format random passenger before converting to JSON for browser
+    random_passenger=list(choice(passengers))
+    if random_passenger[4]=='C':
+        random_passenger[4]='Cherbourg, France'
+    if random_passenger[4]=='S':
+        random_passenger[4]='Southampton, England'
+    if random_passenger[4]=='Q':
+        random_passenger[4]='Queenstown, Ireland'
+    if random_passenger[4]=='B':
+        random_passenger[4]='Belfast, Ireland'
+    
+    if random_passenger[6] is None:
+        random_passenger[6] ='N/A'
```

Comment:
if ticket number was null the browser didn't know how to format it. Here is the simple 'if' logic to fix it